### PR TITLE
Readme translated to english

### PR DESCRIPTION
**Pull request by MaanavD**
English is my primary language, but here's my hand at a translation of my own!

### CODE REVIEW

**Comment by Pacane** [comment]
Hey, that's great work. Thanks!

**Comment by Pacane** [comment]
Bullet points are f*ked up though

**Comment by emileber** [comment]
This is a dupe of #70, please ask before making a PR.

File: README.md [hunk 1]
```diff
@@ -9,17 +9,18 @@
...
 
-filtrez vos recherches par:
-* Date de présentation
-* Distance
-* Prix
 
-Découvrez de nouveau artiste en vous amusant et développez votre CULTURE humoristique!
-Ajoutez des spectacles à vos favoris!
+Filter your searches by:
+*The date of the show
```
Comment (emileber) [inline README.md]: There's a space missing after each `*`.

**Review by emileber — CHANGES_REQUESTED** [review]

**Review by Pacane — APPROVED** [review]

**Pull request merged**